tests/config.yaml

```diff
@@ -34,9 +34,11 @@ bootstrap:
     eligibility-confidence-param: '5'
     eligibility-epoch-offset: '0'
     genesis-active-size: '50'
+    executable-path: '/bin/go-spacemesh'
+  deployment_type: deployment
 client:
-  image: 'spacemeshos/go-spacemesh:develop'
-  command: '/bin/go-spacemesh'
+  image: 'spacemeshos/go-spacemesh-dev:harness'
```

Comment:
dont push these changes. image should stay develop

Reply:
fixed in 704b0c0b747b960ba1d5920e85d81182eaa660a8